test/__mocks__/audio-effects.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env jest */
+export default class MockAudioEffects {
+    static get effectTypes () { // @todo can this be imported from the real file?
```

Comment:
Ack it seems that the `jest.mock` does something globally that stops this file from being allowed to import the actual AudioEffects... I would need to dig deeper into jest to figure out the right way for this.